plugins/woocommerce/client/blocks/assets/js/blocks/cart/test/editor-integration.tsx

```diff
@@ -10,6 +10,23 @@ import { previewCart } from '@woocommerce/resource-previews';
 import { dispatch } from '@wordpress/data';
 import { CART_STORE_KEY as storeKey } from '@woocommerce/block-data';
 
+// Mock problematic dependencies that cause SVG parsing issues
+jest.mock( '@automattic/tour-kit', () => ( {
+	__esModule: true,
+	default: () => null,
+	TourKit: () => null,
+} ) );
+
+jest.mock( 'gridicons/svg-sprite/gridicons.svg', () => 'mocked-svg', {
+	virtual: true,
+} );
+
+jest.mock( '@automattic/components', () => ( {
+	__esModule: true,
+	Card: ( { children }: { children: React.ReactNode } ) => children,
+	Gridicon: () => null,
+} ) );
```

Comment:
Not sure what exactly a problem you had, but I recently faced issue with SVG in integration tests. [I added a stub for SVG and more](https://github.com/woocommerce/woocommerce/pull/60060/files#diff-b903200b31ffb2fb929a9ce5b3af76fe50440a65ead1a4fd1b27959b396f9a31), can you verify you still need this?